operator/install-result.adoc

```diff
@@ -0,0 +1,88 @@
+= Troubleshooting
+:toc: left
+:toclevels: 5
+:source-highlighter: rouge
+:docinfo: shared
+:docinfodir: ../common
+
+
+== Installing Result
+
+ To install the results using Operator, please follow these instructions:
+
+1. Create a database user and password in the `openshift-pipelines` namespace:
++
+[source,bash]
+----
+$ export NAMESPACE="openshift-pipelines"
+$ oc create secret generic tekton-results-postgres --namespace=${NAMESPACE} --from-literal=POSTGRES_USER=result --from-literal=POSTGRES_PASSWORD=$(openssl rand -base64 20)
+----
+
+2. Generate a new self-signed certificate. Alternatively, you can use a certificate provided by the certificate authority (CA).
++
+[source,bash]
+----
+$ openssl req -x509 \
+-newkey rsa:4096 \
+-keyout key.pem \
+-out cert.pem \
+-days 365 \
+-nodes \
+-subj "/CN=tekton-results-api-service.${NAMESPACE}.svc.cluster.local" \
+-addext "subjectAltName = DNS:tekton-results-api-service.${NAMESPACE}.svc.cluster.local"
+----
+
+3. Create a new Transport Layer Security (TLS) secret from the certificate.
++
+[source,bash]
+----
+oc create secret tls -n ${NAMESPACE} tekton-results-tls \
+--cert=cert.pem
+----
+
+4. Optional: Create a Persistent Volume Claim (PVC) if you need to store your logging data.
++
+[source,bash]
+----
+$ cat <<EOF > pvc.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: tekton-logs
+  namespace: tekton-pipelines
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+EOF
+# Apply the above PVC
+$ oc apply -f pvc.yaml
```

Comment:
remove the namespace from yaml and use `${NAMESPACE}` in oc apply command

Reply:
Done